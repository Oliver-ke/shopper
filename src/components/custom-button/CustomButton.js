import React from 'react';
import { CustomButtonContainer } from './customButtonStyle';

const CustomButton = ({ children, ...props }) => {
	return <CustomButtonContainer {...props}>{children}</CustomButtonContainer>;
};

export default CustomButton;
