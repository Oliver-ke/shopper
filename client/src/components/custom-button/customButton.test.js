import React from 'react';
import { shallow } from 'enzyme';
import CustomButton from './CustomButton';

describe('CustomButton component', () => {
	let wrapper;
	beforeEach(() => {
		const mockProps = {
			children: '<p>Save</p>',
		};
		wrapper = shallow(<CustomButton {...mockProps} />);
	});

	it('should render CustomButton component', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
