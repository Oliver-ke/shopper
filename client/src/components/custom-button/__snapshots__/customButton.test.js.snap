// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionItem component should render CollectionItem component 1`] = `ShallowWrapper {}`;

exports[`CustomButton component should render CustomButton component 1`] = `ShallowWrapper {}`;
